certificate-renewal/REPWRITERSPECS/BANNO.CDRENEW.V1.POW

```diff
@@ -346,18 +348,24 @@ DEFINE
  SXFER3EXPIREDATE          = DATE ARRAY(99)
  SXFER3LOC                 = NUMBER ARRAY(99)
  SXFER3SHAREID             = CHARACTER(4) ARRAY(99)
-
  STRANSFERCOUNTMAX         = 99
+
+[* TKainz - added 06/06/24
+*]
+ PARAMINSTALLDATE          = DATE
```

Comment:
I noticed PARAMINSTALLDATE doesn't get initialized. You may want to consider initializing it.

Reply:
Corrected

Reply:
@jordan-keenan Corrected